examples/model_selection/plot_grid_search_stats.py

```diff
@@ -0,0 +1,506 @@
+
+pairwise_bayesian_df = (pd.DataFrame(
+    pairwise_bayesian,
+    columns=['worse_prob', 'better_prob', 'rope_prob']
+    ).round(3))
+
+pairwise_comp_df = pairwise_comp_df.join(pairwise_bayesian_df)
+pairwise_comp_df
+
+
+###############################################################################
+# Using the Bayesian approach we can compute the probability that a model
+# performs better, worse or practically equivalent to another.
+#
+# Results show that the model ranked first by our GridSearch, `rbf`, has
+# approximately a 6.8% chance of being worse than `linear`, and a 1.8% chance
+# of being worse than `poly3`.
+#
+# `rbf` and `linear` have a 43% probability of being practically equivalent,
+# while `rbf` and `poly3` have a 10% chance of being so.
+#
+# Similarly to the conclusions obtained using the Frequentist approach, all
+# models have a 100% probability of being better than `poly2`, and none have a
+# practically equivalent performance with the latter.
+
+
+###############################################################################
+# Take-home messages
+# ------------------
+# - When statistically comparing the performance of two models evaluated in
+#   GridSearchCV, it is necessary to correct the calculated variance that could
+#   be underestimated given that the scores of the models are not independent
+#   from each other.
+# - A Frequentist approach that uses a (variance-corrected) paired t-test can
+#   tell us if the performance of one model is better that another with a
+#   degree of certainty above chance.
+# - A Bayesian approach can provide the probabilities of one model being
+#   better, worse or practically equivalent than another. It can also tell us
+#   how confident we are of knowing that the true differences of our models
+#   fall under a certain range of values.
+# - If multiple models are statistically compared, a multiple comparisons
+#   correction is needed when using the Frequentist approach.
+
+
+###############################################################################
+# References:
+# ___________
+# .. [1] Dietterich, T. G. (1998). Approximate statistical tests for comparing
+#        supervised classification learning algorithms. Neural computation,
+#        10(7).
+# .. [2] Nadeau, C., & Bengio, Y. (2000). Inference for the generalization
+#        error. In Advances in neural information processing systems.
+# .. [3] Bouckaert, R. R., & Frank, E. (2004). Evaluating the replicability of
+#        significance tests for comparing learning algorithms. In Pacific-Asia
+#        Conference on Knowledge Discovery and Data Mining.
+# .. [4] Benavoli, A., Corani, G., Demšar, J., & Zaffalon, M. (2017). Time for
+#        a change: a tutorial for comparing multiple classifiers through
+#        Bayesian analysis. The Journal of Machine Learning Research, 18(1).
+#           - See the Python library that accompanies this paper
+#             `here <https://github.com/janezd/baycomp>`_.
```

Comment:
I would add an empty line at the end for pep8 9https://www.flake8rules.com/rules/W391.html)